Source/WebCore/page/ChromeClient.h

```diff
@@ -241,7 +241,7 @@ class ChromeClient {
     // should not be shown to the user until the ChromeClient of the newly
     // created Page has its show method called.
     // The ChromeClient should not load the request.
-    virtual RefPtr<Page> createWindow(LocalFrame&, const String& openedMainFrameName, const WindowFeatures&, const NavigationAction&) = 0;
+    virtual RefPtr<Page> createWindow(LocalFrame&, const String& openedMainFrameName, const WindowFeatures&, const NavigationAction&, std::optional<Ref<Document>> requester = std::nullopt) = 0;
```

Comment:
Again, this is the wrong type everywhere. `std::optional<Ref<T>>` is not really a thing in WebKit